package.json

```diff
@@ -93,10 +93,12 @@
 		"morgan": "^1.10.0",
 		"multer": "^1.4.5-lts.1",
 		"murmurhash-js": "^1.0.0",
+		"mysql": "^2.18.1",
 		"node-2fa": "^2.0.3",
 		"node-fetch": "^2.6.12",
 		"node-os-utils": "^1.3.7",
 		"nodemailer": "^6.9.4",
+		"pg": "^8.11.3",
```

Comment:
shouldn't we remove these, bit annoying to install both every time u install the server

Reply:
I thought we agreed on having these? Kinda needed for prod anyways, unless I hack them in somehow and use a second lock file...

Reply:
Wait, I think we'd agreed on putting them in `optionalDependencies`, no?